src/ItemManager.js

```diff
@@ -0,0 +1,52 @@
+
+class ItemManager {
+    constructor() {
+        this.tasks = [];
+        this.pokemonClient = new PokemonClient();
+    }
+
+    async handleTask(task) {
+        let taskSplit = task.split(",");
+        if (isNaN(taskSplit[0])) {
+            return [[task], false];
```

Comment:
very confusing to read such a return value, I have no idea what false stands for, I need to search for the usages to understand that